website/content/docs/concepts/dev-server.mdx

```diff
@@ -54,3 +54,25 @@ If you're new to Vault, you may want to pick up with [Your First Secret](/intro/
 
 In addition to experimentation, the dev server is very easy to automate
 for development environments.
+
+
+## Quickstart Option
+
+If you initiate a "dev" server with the flag `-dev-quickstart`
+(e.g. `vault server -dev -dev-quickstart`) a suite of options are enabled to explore
+a configured Vault server.
+- **Identity System** - The server will be automatically initialized
+  with an identity entity and an associated `userpass` authentication method for
+  exploring the identity system. The entity has a policy associated with it to enable
+  reading, writing and deleting secrets.
+- **AppRole Authentication** - The server will be automatically initialized
+  with an `approle` authentication method for exploring a common method to
+  authenticate applications.
+- **Sample Secret** - The server will be automatically initialized
+  with a secret stored in the KVv2 secrets engine.
```

Comment:
Would it maybe be worth adding the secret name/relevant CLI command here too so people have another quick reference besides the startup banner?

Reply:
Yeah, I think that's a great addition! I'll update this with commands and some more details about the resources.